crates/node/src/service/l1.rs

```diff
@@ -52,13 +52,13 @@ impl L1SyncService {
             None
         };
 
-        let gas_price_sync_enabled = authority && !config.gas_price_sync_disabled;
+        let gas_price_sync_enabled = authority && config.gas_price.is_none();
```

Comment:
so as per the new changes, we can either set some values to the gas price, like it can be zero or it can be some other fix fees, so in case it's fixed we don't want to call the gas sync worker, so hence if we have the authority enabled and we have nothing set to gas price, we will call the gas price worker

does it make sense, sorry for not providing the enough information beforehand

Reply:
Great makes sense! Maybe add that in a comment for future readability :)

Reply:
resolved